Makefile

```diff
@@ -76,6 +76,7 @@ codegen: ## Generate code. Must be run if changes are made to ./pkg/apis/...
 		crd \
 		paths="./pkg/..." \
 		output:crd:artifacts:config=charts/karpenter/crds
+	go run hack/code/prices_gen.go -- pkg/cloudprovider/aws/zz_generated.pricing.go
```

Comment:
This makes price generation part of the `verify` target, which may not be ideal. The on-demand prices don't update often, but the spot prices do so this will result in some churn on the static price list.  I could also just move the price generation into the release target.